conf/ansible/inventory/group_vars/all.yaml

```diff
@@ -22,3 +22,27 @@ cron:
   http_proxy:
   https_proxy:
   no_proxy:
+
+AWSELBAccountIDs:
```

Comment:
Please rename so key name format is consistent with the other keys on the other YAML files.

Reply:
I have lower cased the name so it aligns better with the naming. It is now "awselbaccountids"

Reply:
Do the other keys have underscore separators?

Reply:
I cannot supply dashs as ansible raises an exception, hence why the underscore is substituted in the jinja2 template

Reply:
What I meant is that when you check the other keys on the various YAML configs, do you see them as `abc123xyz` ? or does each segment get separated in the fashion of `abc_123_xyz` ? Because the name you ended up with is `awselbaccountids`, is that consistent with the other keys naming convention? Should you use `aws_elb_account_ids` instead?

Reply:
I have renames the list and variables accordingly to match the naming convention